superset-frontend/src/explore/controlUtils.js

```diff
@@ -66,22 +66,25 @@ export function findControlItem(controlPanelSections, controlKey) {
   );
 }
 
-export const getControlConfig = memoizeOne(function getControlConfig(
-  controlKey,
-  vizType,
-) {
+const getMemoizedControlConfig = memoizeOne(
+  (controlKey, controlPanelConfig) => {
+    const {
+      controlOverrides = {},
+      controlPanelSections = [],
+    } = controlPanelConfig;
+
+    const control = expandControlConfig(
+      findControlItem(controlPanelSections, controlKey),
+      controlOverrides,
+    );
+    return control?.config || control;
+  },
+);
+
+export const getControlConfig = function getControlConfig(controlKey, vizType) {
```

Comment:
had to change where the memoization was happening, so that it calculates a new value when the plugin loads